service_flask.py

```diff
@@ -8,6 +8,7 @@
 
 from src.Server import Server
 from flask import Flask, url_for, request, render_template
+import time, datetime
```

Comment:
is time module used?

Reply:
time is not used, taken that out. Datetime still there